### PR TITLE
Add permission on DomainContext

### PR DESCRIPTION
**Pull request by tnagorra**
Depends on https://github.com/idmc-labs/helix-server/pull/56

### CODE REVIEW

File: src/components/EntryForm/index.tsx [hunk 1]
```diff
@@ -53,6 +54,7 @@ import {
     EntryQueryVariables,
 } from '#generated/types';
 
+import route from '../../Root/App/Multiplexer/route';
```
Comment (AdityaKhatri) [inline src/components/EntryForm/index.tsx:57]: Use absolute path

File: src/views/ReviewEntry/index.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ import UrlPreview from '#components/UrlPreview';
 import { FormValues, Attachment, Preview } from '#components/EntryForm/types';
 import { PartialForm } from '#types';
 
+import route from '../../Root/App/Multiplexer/route';
```
Comment (AdityaKhatri) [inline src/views/ReviewEntry/index.tsx:19]: Use absolute import

File: src/utils/common.ts [hunk 3]
```diff
@@ -23,3 +23,22 @@ export function isValidUrl(url: string | undefined): url is string {
     const sanitizedUrl = url.replace(rege, 'localhost.com');
     return isValidRemoteUrl(sanitizedUrl);
 }
+
+export function listToMap<T, K extends string | number, V>(
```
Comment (AdityaKhatri) [inline src/utils/common.ts:27]: Whats wrong with listToMap from fujs?
Move this to fujs or rename or add note

Comment (tnagorra) [reply src/utils/common.ts:27]: It's not typesafe enough. This one may break the existing functionality so, it' not wise to move it to fujs yet.

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**